code/mathematical_algorithms/src/factorial/factorial.ex

```diff
@@ -1,7 +1,7 @@
 defmodule Factorial do 
   def factorial(0), do: 1
```

Comment:
Same here

Reply:
Yep

```elixir
def factorial(n) when n < 0, do: 1
```

Reply:
Yes, it should. You should probably handle all cases.